package.json

```diff
@@ -35,7 +35,7 @@
     "gelf-stream": "^1.1.1",
     "hot-shots": "^4.3.1",
     "js-yaml": "^3.8.1",
-    "limitation": "^0.1.9",
+    "limitation": "git+https://github.com/arlolra/limitation#cleanup",
```

Comment:
@arlolra Ye, we know, shame on us, there's no tests for `service-runner`, but as an excuse I would say it would be extremely hard to write unit tests for this..